client/index.js

```diff
@@ -58,7 +57,7 @@ addFilter(
 		} );
 
 		pages.push( {
-			container: OnboardingExperimentPage,
+			container: Onboarding,
```

Comment:
_Nitpick_: To keep it consistent we can name it `OnboardingPage` rather than `Onboarding`